source/browseMode.py

```diff
@@ -2001,6 +2041,129 @@ def _iterNotLinkBlock(self, direction="next", pos=None):
 				yield TextInfoQuickNavItem("notLinkBlock", self, textRange)
 			item1=item2
 
+	STYLE_ATTRIBUTES = frozenset([
+		'background-color',

```

Comment:
I know this is a pain, but could you please use double quotes consistently rather than mixed double and single quotes? NVDA uses double quotes by default